packages/vats/src/core/chain-behaviors.js

```diff
@@ -295,7 +295,7 @@ export const makeChainStorage = async ({
   const bridgeManager = await bridgeManagerP;
   if (!bridgeManager) {
     console.warn('Cannot support chainStorage without an actual chain.');
-    chainStorageP.resolve(undefined);
+    chainStorageP.resolve(null);
```

Comment:
why change `undefined` to `null`?

You were considering an in-memory chainStorage; that didn't work out?